cfn.yaml

```diff
@@ -31,7 +31,7 @@ Resources:
           PolicyDocument:
             Statement:
             - Effect: Allow
-              Action: s3:PutObject
+              Action: [s3:PutObject, s3:PutObjectAcl]
```

Comment:
Can't `grantRead` rights unless we have the `s3:PutObjectAcl` permission available to us.